whitesource/util.py

```diff
@@ -34,9 +38,23 @@ def create_whitesource_client(
         wss_endpoint=ws_config.wss_endpoint(),
         ws_creds=ws_config.credentials(),
         product_token=ws_config.product_token(),
+        requester_mail=ws_config.requester_mail(),
     )
 
 
+def _mk_ctx():
+    scanned = 1
```

Comment:
should this not rather be initialised w/ 0?

Reply:
The `scanned` counter is for logging purposes only as its indexing how many artifacts are scanned already.
Initializing with `1` is more straight forward and increases readability.